### PR TITLE
Update date for SPACE 2023, disable scroll bouncing, make register li…

### PR DESCRIPTION
**Pull request by alexzhang1618**
- Update date in SEO and description for SPACE 2023
- Disable scroll bouncing through globals.css (this would show white bounds if you scroll too high/low)
- Make the register link open in a new tab (previously, it would overwrite the original tab)
- Rename Participating Companies to indicate that this was for SPACE 2022

### CODE REVIEW

**Review by raymosun — DISMISSED** [review]
Looks good 👍

File: src/components/SEO/index.tsx [hunk 1]
```diff
@@ -20,10 +20,7 @@ const SEO: React.FC = () => {
       {/* title to display for the specific link being shared */}
       <meta property="og:title" content="SPACE 2022" />
```
Comment (SheepTester) [inline src/components/SEO/index.tsx]: Is this supposed to be changed too

File: src/styles/globals.scss [hunk 2]
```diff
@@ -1,3 +1,4 @@
-*{
-    font-family: 'Rubik', sans-serif !important;
-}
\ No newline at end of file
+* {
+  font-family: 'Rubik', sans-serif !important;
+  overscroll-behavior: none;
```
Comment (SheepTester) [inline src/styles/globals.scss:3]: Is there a reason why you're overriding native scroll effects

File: src/components/SEO/index.tsx (hunk 1 above)
Comment (alexzhang1618) [reply src/components/SEO/index.tsx]: Oops good catch, I just updated the SEO stuff

File: src/styles/globals.scss (hunk 2 above)
Comment (alexzhang1618) [reply src/styles/globals.scss:3]: This is what it looks like with scroll bouncing enabled, so I'm overriding it to stop it from happening

https://github.com/acmucsd/space-2022/assets/27360825/e9e501e9-1493-4ecb-a915-4d4ca6f00076

Comment (SheepTester) [reply src/styles/globals.scss:3]: ah I see, getting the gradient out there seems involved

**Pull request merged**